tests/unit/test_charm/test_charm.py

```diff
@@ -327,3 +338,56 @@ def test_parca_external_store_relation(context, base_state):
         f"--insecure=false "
         f"--mode=scraper-only",
     )
+
+
+def test_self_profiling_no_endpoint_relation(context, base_state):
+    # verify that the scrape config contains the self-scraping job
+    expected_scrape_config = [
+        {
+            "job_name": "parca",
+            "relabel_configs": [
+                {
```

Comment:
if this were in a global somewhere we wouldn't have to copy-paste it. it's unlikely to change it so I'd do that...